### PR TITLE
Update signer standards overview with split standards

### PR DESCRIPTION
**Pull request by frederikrothenberger**
It removes the repeated sequence diagrams to reduce the work required to keep the overview up to date.

This PR also contains smallish updates on the readme to better reflect the state of things.

### CODE REVIEW

**Review by plitzenberger — APPROVED** [review]
lgtm and I feel honored 🙏

File: README.md [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
+| ICRC-21  | Canister Call Consent Messages                  | [Frederik Rothenberger](https://github.com/frederikrothenberger) | [Draft Standard](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_21_consent_msg.md)                   | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
+| ICRC-25  | Signer Interaction Standard                     | [Philipp Litzenberger](https://github.com/plitzenberger)         | [Draft Standard](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/icrc_25_signer_interaction_standard.md)   | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
```
[the remark this answers is not in the document]
Comment (frederikrothenberger) [reply README.md:11]: Yes, that's a nice idea. Let me quickly do that. 🙂

[the remark this answers is not in the document]
Comment (plitzenberger) [reply README.md]: Ok, shall I take care of it?

[the remark this answers is not in the document]
Comment (frederikrothenberger) [reply README.md]: It is done, unless you would like further changes to the readme?

**Pull request merged**